src/Components/OATPropertyEditor/FormRootModelDetails.tsx

```diff
@@ -220,6 +242,44 @@ export const FormUpdateProperty = ({
         setIsAMultiLanguageDescriptionEmpty(hasEmptyValues);
     }, [multiLanguageSelectionsDescription]);
 
+    const lookUpStoredMetadata = () => {
+        // Check if there is metadata for the model, if so update fileName and directoryPath
+        if (modelsMetadata) {
+            const modelMetadata = modelsMetadata.find(
+                (modelMetadata: any) => modelMetadata['@id'] === model['@id']
+            );
+            if (modelMetadata) {
+                setFileName(modelMetadata.fileName);
+                setDirectoryPath(modelMetadata.directoryPath);
+                return true;
+            }
+            return false;
```

Comment:
Is this required? Won't it just fall through to the return false on the next line?